addons/payment/models/res_partner.py

```diff
@@ -17,3 +17,16 @@ def _compute_payment_token_count(self):
         mapped_data = dict([(payment['partner_id'][0], payment['partner_id_count']) for payment in payment_data])
         for partner in self:
             partner.payment_token_count = mapped_data.get(partner.id, 0)
+
+    def _payment_methods(self):
```

Comment:
it seems to be a bit of a mess to get payment methods for a partner, i don't see why this needs to exist since you have a method on payment.acquirer which takes the partner as an argument